bigtable/google/cloud/bigtable/row_data.py

```diff
@@ -440,86 +422,78 @@ def __iter__(self):
             except StopIteration:
                 break
 
-            self._counter += 1
```

Comment:
`_counter` is documented in `__init__()` as a count of responses processed, not rows returned (as it seems to be below).

Reply:
I'll update the documentation.  `_counter` affects retries at the row level, not the "chunk" or "response" level.